WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,19 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ border-bottom:1px solid @SeparatorColor@; @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 6px;
```

Comment:
For 16px, you can use `SkinWrapperPadding` variable. It's property to indicate generic spacing.